xdp-root/xdp_root.c

```diff
@@ -37,8 +37,8 @@ static int ifindex_in;
 
 static __u32 xdp_flags;
 
-static const char *prog_root_pass_file = "/sys/fs/bpf/xdp_root_pass_array";
-static const char *prog_root_file = "/sys/fs/bpf/xdp_root_array";
+static const char *prog_root_pass_file = "C:\\leaf\\sys\\fs\\bpf\\xdp_root_pass_array";
+static const char *prog_root_file = "C:\\leaf\\sys\\fs\\bpf\\xdp_root_array";
```

Comment:
Revert.  These are not fileysstem paths.

Reply:
Done